test/managedcluster/providervalidator.go

```diff
@@ -64,6 +64,8 @@ func NewProviderValidator(template Template, clusterName string, action Validati
 		case TemplateAWSStandaloneCP, TemplateAWSHostedCP:
 			resourcesToValidate["ccm"] = validateCCM
 			resourceOrder = append(resourceOrder, "ccm")
+		case TemplateAzureStandaloneCP, TemplateVSphereHostedCP:
+			delete(resourcesToValidate, "csi-driver")
```

Comment:
I see earlier you were setting up a `csi-driver` to be used with Azure, should this test be enabled for Azure then?  Or is the `csi-driver` intentionally only for `hosted`?